src/components/dashboard/CoachDashboard.tsx

```
@@ -308,6 +308,9 @@ function CoachDashboard() {
                       return (
                         <div
                           key={runner.id}
+                          data-testid={`runner-card-${runner.id}`}
+                          role="article"
+                          aria-label={`Runner card for ${runner.full_name || 'User'}`}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<article>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)